share/efuncs.sh

```diff
@@ -1338,6 +1338,76 @@ directory_not_empty()
     find "${dir}" -mindepth 1 -print -quit | grep -q .
 }
 
+opt_usage relative_path<<'END'
+# relative_path takes two paths and figures out the relative path between the two.
+# ex: the relative_path between /home/user/foo/bar and /home/user/baz/buz is
+# ../../baz/buz
```

Comment:
haha...  unclear comment... if bar is a dir the relpath is ../../baz/buz... if bar is a path, the relpath is ../baz/buz.  fortunately, the code does the right thing even  if the comment isn't 100% clear